package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "@ckeditor/ckeditor5-vue",
+  "name": "@ckeditor/ckeditor5-vue2",
   "version": "1.0.3",
   "main": "dist/ckeditor.js",
   "description": "Official Vue.js component for CKEditor 5 – the best browser-based rich text editor.",
```

Should we update the description? cc: @AnnaTomanek

---

Makes sense, thanks for catching this! How about:

```suggestion
  "description": "Official Vue.js 1.x and 2.x component for CKEditor 5 – the best browser-based rich text editor.",
```